app/_kong_plugins/session/index.md

```diff
@@ -121,6 +121,85 @@ present and the incoming request is a logout request, the Session plugin
 returns a 200 before continuing in the plugin run loop, and the request doesn't
 continue to the upstream.
 
+## Kong Manager
+
+To enable sessions authentication, configure the following parameters in `kong.conf`:
+
+```
+enforce_rbac = on
+admin_gui_auth = <set to desired auth type>
+admin_gui_session_conf = {
+    "secret":"$SET_SECRET",
+    "cookie_name":"$SET_COOKIE_NAME",
+    "storage":"$SET_STORAGE",
+    "rolling_timeout":$NUMBER_OF_SECONDS_UNTIL_RENEWAL,
+    "cookie_secure":$SET_DEPENDING_ON_PROTOCOL,
+    "cookie_same_site":"$SET_DEPENDING_ON_DOMAIN",
+    "store_metadata": true
+}
+```
+
+{% table %}
+columns:
+  - title: Attribute
+    key: attribute
+  - title: Description
+    key: description
+rows:
+  - attribute: "`cookie_name`"
+    description: >
+      A name for the cookie. <br> For example, `"cookie_name":"kong_cookie"`
+  - attribute: "`secret`"
+    description: >
+      The secret used in keyed HMAC generation. Although the Session plugin's default is a random string, the `secret` _must_ be manually set for use with Kong Manager since it must be the same across all Kong workers/nodes.
+  - attribute: "`storage`"
+    description: >
+      The location where session data is stored. <br> The default value is `cookie`. It may be more secure if set to `kong`, since access to the database would be required.
+  - attribute: "`cookie_lifetime`"
+    description: >
+      The duration (in seconds) that the session will remain open. <br> The default value is `3600`.
+  - attribute: "`cookie_secure`"
+    description: >
+      Applies the Secure directive so that the cookie may be sent to the server only with an encrypted request over the HTTPS protocol. See [Session Security](#session-security) for exceptions. <br> The default value is `true`.
```

Comment:
```suggestion
      Applies the Secure directive so that the cookie may be sent to the server only with an encrypted request over the HTTPS protocol. See [Session security overview](#session-security-overview) for exceptions. <br> The default value is `true`.
```